### PR TITLE
[1.22] cgroup v2

### PR DESCRIPTION
**Pull request by giuseppe**
Link to the enhancement: https://github.com/kubernetes/enhancements/issues/2254

Signed-off-by: Giuseppe Scrivano <gscrivan@redhat.com>

### CODE REVIEW

**Comment by PI-Victor** [comment]
thank you for opening this!

/assign @chrisnegus 
/sig node

**Comment by chrisnegus** [comment]
Hi @giuseppe :wave: I'm the 1.22 release docs shadow assigned to track this PR. Please take a look at [Documenting for a release - PR Ready for Review](https://kubernetes.io/docs/contribute/new-content/new-features/#pr-ready-for-review) to get your PR ready for review before Tuesday 20th of July, 11:59 PM PDT. Thank you!

**Comment by mrunalp** [comment]
@chrisnegus Giuseppe is out on vacation. I will look at this early next week and open a new PR if needed.

**Comment by giuseppe** [comment]
@chrisnegus @mrunalp @saschagrunert I had a look at the documentation, I am not sure there is anything we need to add for cgroup v2 as there are no user visible changes

**Comment by saschagrunert** [comment]
I think we could add a paragraph about cgroups v1 vs v2 to https://kubernetes.io/docs/setup/production-environment/container-runtimes/ mentioning:

- Outlining the benefits of using v2 instead of v1
- How to configure a node using cgroups v2
- That container runtimes (as well as OCI runtimes) have to support this as well
- That nothing should change on the user-experience when switching to v2

**Comment by PI-Victor** [comment]
> Hi @giuseppe 👋 I'm the 1.22 release docs shadow assigned to track this PR. Please take a look at [Documenting for a release - PR Ready for Review](https://kubernetes.io/docs/contribute/new-content/new-features/#pr-ready-for-review) to get your PR ready for review before Tuesday 20th of July, 11:59 PM PDT. Thank you!

Hey folks, friendly reminder that today is the deadline for PRs ready for review for docs, please make sue that this PR is ready to review by the end of the day, thank you!

**Review by saschagrunert — APPROVED** [review]
/lgtm

File: content/en/docs/setup/production-environment/container-runtimes.md [hunk 1]
```diff
@@ -57,6 +57,38 @@ If you have automation that makes it feasible, replace the node with another usi
...
+```shell
+# dnf install -y grubby && \
+  sudo grubby \
+  --update-kernel=ALL \
+  --args=”systemd.unified_cgroup_hierarchy=1"
+```
+
+To apply the configuration, it is necessary to reboot the node.
+
+There should not be any noticeable difference in the user-experience when switching to cgroup v2, unless
```
Comment (tengqm) [inline content/en/docs/setup/production-environment/container-runtimes.md:87]:
```suggestion
There should not be any noticeable difference in the user experience when switching to cgroup v2, unless
```

File: content/en/docs/setup/production-environment/container-runtimes.md [hunk 2]
```diff
@@ -57,6 +57,38 @@ If you have automation that makes it feasible, replace the node with another usi
...
+accessing directly the cgroup file system, either on the node or from within the containers.
```
Comment (tengqm) [inline content/en/docs/setup/production-environment/container-runtimes.md:88]:
```suggestion
users are accessing the cgroup file system directly, either on the node or from within the containers.
```

**Comment by giuseppe** [comment]
@tengqm thanks for the review.  Addressed the comments and pushed a new version

**Pull request merged**